packages/kit/src/template.ts

```diff
@@ -180,6 +180,8 @@ export function resolveLayerPaths (dirs: LayerDirectories, projectBuildDir: stri
     nuxt: [
       join(relativeSrcDir, '**/*'),
       join(relativeModulesDir, `*/runtime/**/*`),
+      join(relativeRootDir, `test/**/*`),
+      join(relativeRootDir, `tests/**/*`),
```

Comment:
this is slightly safer:

```suggestion
      join(relativeRootDir, `test/nuxt/**/*`),
      join(relativeRootDir, `tests/nuxt/**/*`),
```

... and we can document in the testing.md file how to add other directories if nuxt runtime environment is located elsewhere.